core-common/src/main/java/org/mapstruct/MapperConfig.java

```diff
@@ -80,7 +80,7 @@
      *
```

Comment:
Yes, I'll update the javadoc. 
Although the restriction that only interfaces or abstract classes are valid config sources now is not something that is currently technically required or checked. Let's do this in a seperate issue. OK?